examples/dao/scripts/deploy.js

```diff
@@ -0,0 +1,101 @@
+const { types } = require('@algo-builder/web');
+const { fundAccount, executeTx } = require('./run/common/common.js');
+
+const { accounts, getDepositLsig, getDAOFundLsig } = require('./run/common/accounts');
+
+async function run (runtimeEnv, deployer) {
+  const { creator, proposer, voterA, voterB } = accounts(deployer);
+
+  // fund accounts
+  await fundAccount(deployer, [creator, proposer, voterA, voterB]);
+
+  // Create DAO Gov Token
+  const asaInfo = await deployer.deployASA('gov-token', { creator: creator });
+  console.log(asaInfo);
+
+  // DAO App initialization parameters
+  const deposit = 15; // deposit required to make a proposal
+  const minSupport = 5; // minimum number of yes power votes to validate proposal
+  const minDuration = 1 * 60; // 1min (minimum voting time in number of seconds)
+  const maxDuration = 5 * 60; // 5min (maximum voting time in number of seconds)
+  const url = 'www.my-url.com';
+
+  const appArgs = [
+    `int:${deposit}`,
+    `int:${minSupport}`,
+    `int:${minDuration}`,
+    `int:${maxDuration}`,
+    `str:${url}`
+  ];
+  const templateParam = { ARG_GOV_TOKEN: asaInfo.assetIndex };
+  // Create Application
+  const daoAppInfo = await deployer.deployApp(
+    'dao-app-approval.py',
+    'dao-app-clear.py', {
+      sender: creator,
+      localInts: 9,
+      localBytes: 7,
+      globalInts: 4,
+      globalBytes: 2,
+      appArgs: appArgs
+    }, {}, templateParam);
+  console.log(daoAppInfo);
+
+  // fund lsig's
+  await Promise.all([
+    deployer.fundLsig('deposit-lsig.py',
+      { funder: creator, fundingMicroAlgo: 2e6 }, {},
```

Comment:
```suggestion
      { funder: creator, fundingMicroAlgo: 2e6 }, {},  // 2 algo
```

Reply:
Why this needs 2 Algo? Can we decrease it?

Reply:
decreased to 1.